yt/yt/tests/integration/queues/test_queue_agent.py

```diff
@@ -3149,7 +3289,7 @@ def test_alert_combinations(self):
         alerts = queue_agent_orchid.get_queue_orchid("primary://tmp/q").get_alerts()
         alerts.assert_matching("queue_agent_queue_controller_static_export_failed", text="has no child with key", attributes={"export_name": "first"})
         alerts.assert_matching("queue_agent_queue_controller_static_export_failed", text="export_destination", attributes={"export_name": "second"})
-        alerts.assert_matching("queue_agent_queue_controller_trim_failed", text="with no vital consumers")
+        alerts.assert_matching("queue_agent_queue_controller_trim_failed", text="Not trimming partition 0 since export")
```

Comment:
This doesn't seem like the right thing to check.

I don't think we should make the effort to distinguish failed exports from non-failed disabled ones in trimming. Both of these cases should lead to nothing being trimmed.

Failed exports will cause their own alert. 

You can change this part of the test to set an incorrect export config and check that we have bot a misconfiguration alert and a trimming alert (since we can't trim while exports are misconfigured).

The intent of the test was just to test alerts in multiple categories at once, since it is a new feature.